src/main/java/client/ZipCodeClient.java

```diff
@@ -0,0 +1,45 @@
+package client;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.http.HttpResponse;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static data.Constants.ZIPCODES_EXPAND_RESOURCE;
+import static data.Constants.ZIPCODES_RESOURCE;
+
+public class ZipCodeClient {
+    private static HttpResponse response;
+
+    public int getResponseStatusCode() {
```

Comment:
bad class implementation when some methods depend on calling other methods. What happens if some developer decides to use your class and writes the following code?
```
ZipCodeClient client = new ZipCodeClient();
int status = client.getResponseStatusCode();
```